walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/socials.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Socials</title>
+
+
+  <style>
+    ul{
```

Comment:
move CSS to new style file instead of inlining inside html file

Reply:
Done!